### ISSUE TITLE
[Feature Request]: Please support "--use-cpu gfpgan codeformer" Start Prompt

### ISSUE THREAD

**Issue by HyunJae5463** [lllyasviel/stable-diffusion-webui-forge#565]
### Is there an existing issue for this?

- [X] I have searched the existing issues and checked the recent builds/commits

### What would your feature do ?

Enables Face Restoration to run on CPU, since IPEX has issues with it and it only works if it runs via the CPU

### Proposed workflow

Just want "--use-cpu gfpgan codeformer" starting prompt to work like in A1111 or SD.Next so either face restoration model runs on the CPU.

**Comment by redrum-llik** [comment]
Indeed, A1111 lets user specify device on a separate basis - either GUI option (like the one for RNG) or a cmd argument would be great to have. For now, patching `devices.py` to override the device is feasible as a workaround.

**Comment by mr-september** [comment]
+1 this is required for both AMD and Intel users. Well documented issue:

https://github.com/lllyasviel/stable-diffusion-webui-forge/issues?q=codeformer
